src/redux/sagas/LoginSaga/loginSaga.js

```diff
@@ -12,8 +11,8 @@ function* loginWorker({ payload }) {
       payload: { user: response.data.user },
     });
     const { token } = response.data.user;
-    setUserToken(token);
-    setCurrentUser(response.data.user);
+    localStorage.setItem('accessToken', JSON.stringify(token));
```

Comment:
Why was this part changed to stop using the `setUserToken` function?

Reply:
Hello,
we did this because a helper function was set up in utils from which the application is able to call from.

Regards
Steven Ennis